Documentation/helm-ceph-cluster.md

```diff
@@ -48,22 +48,65 @@ helm install --create-namespace --namespace rook-ceph rook-ceph-cluster \
 
 The following tables lists the configurable parameters of the rook-operator chart and their default values.
 
-| Parameter             | Description                                                          | Default     |
-| --------------------- | -------------------------------------------------------------------- | ----------- |
-| `operatorNamespace`   | Namespace of the Rook Operator                                       | `rook-ceph` |
-| `configOverride`      | Cluster ceph.conf override                                           | <empty>     |
-| `toolbox.enabled`     | Enable Ceph debugging pod deployment. See [toolbox](ceph-toolbox.md) | `false`     |
-| `toolbox.tolerations` | Toolbox tolerations                                                  | `[]`        |
-| `toolbox.affinity`    | Toolbox affinity                                                     | `{}`        |
-| `monitoring.enabled`  | Enable Prometheus integration, will also create necessary RBAC rules | `false`     |
-| `cephClusterSpec.*`   | Cluster configuration, see below                                     | See below   |
+| Parameter              | Description                                                          | Default     |
+| ---------------------- | -------------------------------------------------------------------- | ----------- |
+| `operatorNamespace`    | Namespace of the Rook Operator                                       | `rook-ceph` |
+| `configOverride`       | Cluster ceph.conf override                                           | <empty>     |
+| `toolbox.enabled`      | Enable Ceph debugging pod deployment. See [toolbox](ceph-toolbox.md) | `false`     |
+| `toolbox.tolerations`  | Toolbox tolerations                                                  | `[]`        |
+| `toolbox.affinity`     | Toolbox affinity                                                     | `{}`        |
+| `monitoring.enabled`   | Enable Prometheus integration, will also create necessary RBAC rules | `false`     |
+| `cephClusterSpec.*`    | Cluster configuration, see below                                     | See below   |
+| `ingress.dashboard`    | Enable an ingress for the ceph-dashboard                             | `{}`        |
+| `cephBlockPools.[*]`   | A list of CephBlockPool configurations to deploy                     | See below   |
```

Comment:
How about in the "Before installing" section above (starts on line 28), you also add a line that explains that the default is to create one pool, one filesystem, an object store, and their corresponding storage classes?